website/src/app/layout.tsx

```diff
@@ -19,7 +19,7 @@ export default function RootLayout({
 }) {
   useEffect(() => {
     document.documentElement.lang = i18next.language;
-  }, [i18next.language]);
+  });
```

Comment:
Why remove the dependency array here ? 

If you do not want the effect to run more that once, add an empty array as dependencies:
 
```suggestion
  }, []);
```